src/pages/404-page/PageNotFound.jsx

```diff
@@ -0,0 +1,13 @@
+import "./PageNotFound.css"; 
+
+const ErrorPage = () => {
+
+    return(
+        <div className="error-container">
+        <h1>404 Error</h1> 
+        <h1>Page Not Found</h1>
+        </div>
```

Comment:
You can add a button to re-direct the user back to home when he visits a 404 page